mdpow/workflows/base.py

```diff
@@ -24,7 +24,7 @@
 import re
 import pandas as pd
 
-from mdpow.workflows import registry
+from ..workflows import registry
```

Comment:
`base.py` is in the same directory as `registry.py`, isn't it?
```suggestion
from . import registry
```
so write it more simply!